external/atomic_queue/include/atomic_queue/atomic_queue.h

```diff
@@ -76,8 +75,8 @@ constexpr T& map(T* elements, unsigned index) noexcept {
 
 ////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
 
-// Implement a "bit-twiddling hack" for finding the next power of 2
-// in either 32 bits or 64 bits in C++11 compatible constexpr functions
+// Implement a "bit-twiddling hack" for finding the next power of 2 in either 32 bits or 64 bits
+// in C++11 compatible constexpr functions. The library no longer maintains C++11 compatibility.
```

Comment:
My concern was this. I'll check but one of my personal targets used to have a C++11 limitation, which I think is lifted now.

Reply:
That being said, as far as I remember, the only issue was some multi-line `constexpr` functions which are just cosmetics. We could very well rewrite these as C++11 compatible constructs, which is what I did at the time but Maxim did not want to constrain his library in the end.